### DISCUSSION TITLE
docker中alist无法启动（data.db问题）

### DISCUSSION THREAD

**Discussion by myhome-git** [AlistGo/alist#6651] [General]
使用环境为debian11，安装了docker，之前一直正常，2024年6月24日突然无法访问，通过docker后台看到alist的状态是正在重启，并且始终无法启动，而后手动删除data.db后启动成功，最后确认是data.db文件（sqlite3）的问题，但是之前存储了很多配置都在data.db中，这可麻烦了

**Comment by pongfcnkl** [comment]
手动获取sqlite里的数据文件吧
